package.json

```diff
@@ -35,7 +35,7 @@
     "fast-html-parser": "^1.0.1",
     "google-translate-api": "^2.3.0",
     "idiotic-api": "^0.2.8",
-    "klasa": "github:dirigeants/klasa",
+    "klasa": "github:dirigeants/klasa#SGv2.2.0",
```

Comment:
if SG v2.2 gets merged this wont work.

Reply:
Yes but... if you're going to merge this into rewrite when SGv2.2 is merged then it creates more work for you when you could just remove the branch and have it ready to be merged.

Reply:
Changing a dependency isn't more work. I have it this way so I can test.

Reply:
Making a commit = work. but okay